changelog/README.rst

```diff
@@ -0,0 +1,40 @@
+This directory contains "newsfragments" which are short files that contain a small **ReST**-formatted
+text that will be added to the next ``CHANGELOG``.
+
+The ``CHANGELOG`` will be read by **users**, so this description should be aimed to pytest users
+instead of describing internal changes which are only relevant to the developers.
+
+Make sure to use full sentences in the **present tense** and use punctuation, examples::
+
+    Improve Guild.create_text_channel returning the channel type.
+
+    Command syncing now uses logging instead of print.
+
+Each file should use the following naming: ``<ISSUE>.<TYPE>.rst``, where
+``<ISSUE>`` is an issue number, and ``<TYPE>`` is one of:
+
+* ``feature``: new user facing features, like new command-line options and new behavior.
+* ``bugfix``: fixes a bug.
+* ``doc``: documentation improvement, like rewording an entire session or adding missing docs.
+* ``removal``: feature deprecation.
+* ``breaking``: a change which may break existing suites, such as feature removal or behavior change.
+* ``misc``: fixing a small typo or internal change that might be noteworthy.
+
+So for example: ``123.feature.rst``, ``456.bugfix.rst``.
+
+If your PR fixes an issue, use that number here. If there is no issue,
+then after you submit the PR and get the PR number you can add a
+changelog using that instead.
+
+If you are not sure what issue type to use, don't hesitate to ask in your PR.
+
+``towncrier`` preserves multiple paragraphs and formatting (code blocks, lists, and so on), but for entries
+other than ``features`` it is usually better to stick to a single paragraph to keep it concise.
+
+You can also run ``nox -s docs`` to build the documentation
```

Comment:
May want to recommend `task docs` instead? Not sure.

Reply:
In that case, we might want to recommend `nox -Rs docs` instead of `nox -s docs` here?

Reply:
We can recommend elsewhere that `-R` can be used to reuse the venv, but anyone who already has a docs venv will need to recreate without the `-R` to install sphinxcontrib-towncrier.